packages/examples/packages/signature-insights/src/index.ts

```diff
@@ -0,0 +1,105 @@
+import type { OnSignatureHandler } from '@metamask/snaps-sdk';
+import { SeverityLevel, panel, text, row, heading } from '@metamask/snaps-sdk';
+
+// The verifying contract in test dapp e2e tests
+const MALICIOUS_CONTRACT = '0xCcCCccccCCCCcCCCCCCcCcCccCcCCCcCcccccccC';
```

Comment:
Can you explain this a bit more (in a doc comment)?

Also:

```suggestion
// The verifying contract in test dapp e2e tests
const MALICIOUS_CONTRACT = '0xCcCCccccCCCCcCCCCCCcCcCccCcCCCcCcccccccC';

```